test/testautomation_video.c

```diff
@@ -324,7 +324,7 @@ int video_getWindowFlags(void *arg)
     if (window != NULL) {
         actualFlags = SDL_GetWindowFlags(window);
         SDLTest_AssertPass("Call to SDL_GetWindowFlags()");
-        SDLTest_AssertCheck((flags & actualFlags) == flags, "Verify returned value has flags %d set, got: %" SDL_PRIu32, flags, actualFlags);
+        SDLTest_AssertCheck((flags & actualFlags) == flags, "Verify returned value has flags %u set, got: %" SDL_PRIu64, flags, actualFlags);
```

Comment:
`SDL_WindowFlags flags;` is also 64-bit.

Reply:
Use `SDL_PRIu64` for both? Done.

Reply:
Didn't compile for a lot of systems. I'll switch it back to `%u`.

Question is if some compilers are "smart" enough to realize that we aren't using any bits above 32 yet and just respecting the unsigned appendix on the enum? I'm not sure. Need to read up on standard.